src/desktop/open_uri.rs

```diff
@@ -171,6 +171,13 @@ impl OpenFileRequest {
         self
     }
 
+    /// Sets the token that can be used to activate the chosen application.
```

Comment:
I see that you are only using this in your zed PR, please split it in a separate commit so that I could backport it for you otherwise you would have to wait for a major release in few weeks

Reply:
Appreciate the concerns, but I think that Zed could switch to the git repo for now (Zed on Linux is still in development) :slightly_smiling_face: